recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.0.0rc1" %}
+{% set version = "2.0.5" %}
```

Comment:
@wachsylon this is the latest stable one, right? The same one [we have here](https://github.com/conda-forge/cdo-feedstock/blob/main/recipe/meta.yaml#L1). Do you want to update to a latest dev release?

Reply:
I am happy with that version just as well with other versions - but for now lets keep this.

Just as a reminder, you set up the dev channel for me which allows me to specify an installation which links to an additional package named cmor. Since this package is rather big and has many dependencies, it might be a good idea to keep this installation in a separate channel. Also, cdo with cmor has very specifc use cases. however, i did not discuss this with the main developers for a long time.

Reply:
> Just as a reminder, you set up the dev channel for me which allows me to specify an installation which links to an additional package named cmor.

I forgot about that entirely! Also, you should have merge rights here. Do you mind adding yourself to the maintainers list?